dandi/tests/data/dandiarchive-docker/docker-compose.yml

```diff
@@ -36,6 +36,8 @@ services:
       DJANGO_DANDI_WEB_APP_URL: http://localhost:8085
       DJANGO_DANDI_API_URL: http://localhost:8000
       DJANGO_DANDI_JUPYTERHUB_URL: https://hub.dandiarchive.org
+      # Added for testing
+      DJANGO_DANDI_INSTANCE_NAME: DANDI
```

Comment:
might need the other line from 
- https://github.com/dandi/dandi-infrastructure/pull/224/files

identifier for RRID